Atomic/scan.py

```diff
@@ -188,6 +188,17 @@ def gen_containers():
             scan_list = []
             images = self.get_images()
             containers = self.get_containers()
+            all_names = []
```

Comment:
This is designed for the Cockpit UI if the user manually selects all images and containers, then I can check to see if their scan_targets matches all the images and containers listed in their system.